setup.sh

```
@@ -18,25 +19,25 @@ rm godot.zip
 # The zip extracts to a folder named 'templates'. We move its contents.
-mv templates/* ~/.local/share/godot/export_templates/4.6.stable/
+mv templates/* ~/.local/share/godot/export_templates/${GODOT_VERSION}.stable/
 
 # 8. Clean up
 rm -rf templates templates.tpz
 
 # 9. Handle "nothreads" error
 # Some Godot versions/presets look for 'web_nothreads_debug.zip' specifically.
 # If it's missing, we copy the standard debug template to satisfy the builder.
-cd ~/.local/share/godot/export_templates/4.6.stable/
+cd ~/.local/share/godot/export_templates/${GODOT_VERSION}.stable/
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)